tests/dialects/test_postgres.py

```diff
@@ -85,6 +85,20 @@ def test_ddl(self):
                 read="postgres",
             )
 
+    def test_unnest(self):
+        self.validate_all(
+            "SELECT UNNEST(c) FROM t",
+            write={
+                "presto": "SELECT col FROM t CROSS JOIN UNNEST(c) AS _u(col)",
```

Comment:
Should maybe add a hive translation here, too? I think it should become explode in Hive? i.e. `SELECT EXPLODE(c) FROM t`

Reply:
On second look, that might be a different PR